roombasim/pittras/task/land_task.py

```diff
@@ -0,0 +1,36 @@
+
+import numpy as np
+
+import roombasim.config as cfg
+
+from roombasim.ai import Task, TaskState
+
+class LandTaskState:
```

Comment:
This should inherit from `object`, no?

Reply:
I think Python 3 heavily discouraged a class from inheriting from nothing so it should inherit from `object`.

Reply:
In Python 2 you should really inherit from `object`, in Python 3 you don't have to but it doesn't hurt if you do

Reply:
Have we agreed on which python version we are primarily supporting? If we still want compatibility for both python 2 and 3, I think it will be better to have the inheritance explicit.

Reply:
We _need_ support for python 2, python 3 support is up to you guys